solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttp2SolrClient.java

```diff
@@ -88,23 +89,63 @@ public LBHttp2SolrClient(Http2SolrClient httpClient, String... baseSolrUrls) {
     this.httpClient = httpClient;
   }
 
+  public LBHttp2SolrClient(Http2SolrClient httpClient, List<String> baseSolrUrls, Builder builder) {
```

Comment:
should this be 'protected' instead of 'public'?

Reply:
@epugh raising this comment for your review, in case it got lost

Reply:
Okay, check out the latest ocmmit @stillalex  ;-)

Reply:
could you explain a bit? I am not seeing the visibility change on the constructor (I am seeing the builder removal)

Reply:
Humm...  I may be getting confused...    (honestly, if you want to push up a commit, I'd love to see it!).   Right now on `LBHttpSolrClient` we have `protected LBHttpSolrClient(Builder builder)` and that's it...      On `LBHttp2SolrClient` we have the deprecated `public LBHttp2SolrClient(Http2SolrClient httpClient, String... baseSolrUrls) ` and `private LBHttp2SolrClient(Http2SolrClient httpClient, List<String> baseSolrUrls)` methods...